test/fuzz/oss_fuzz_build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
```

Comment:
can you also add Envoy Gateway license headers ?

Reply:
added